examples/etherpad/Nulecule.yaml

```diff
@@ -0,0 +1,104 @@
+# Using etherpad as an example
```

Comment:
Bit confused with this comment ^^. We don't use compose in atomicapp / nulecule. Can you provide an example what you mean?